core/modules/parsers/wikiparser/rules/syslink.js

```diff
@@ -0,0 +1,45 @@
+/*\
+title: $:/core/modules/parsers/wikiparser/rules/syslink.js
+type: application/javascript
+module-type: wikirule
+
+Wiki text inline rule for system tiddler links.
+Can be suppressed preceding them with `~`.
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+exports.name = "syslink";
+exports.types = {inline: true};
+
+exports.init = function(parser) {
+	this.parser = parser;
+	// Regexp to match
+	this.matchRegExp = /~?(?:\$):[^\s<>{}\[\]`|'"\\^~]+(?:\/|\b)/mg;
```

Comment:
Am 10.01.15 15:40, schrieb Tobias Beer - notifications@github.com:
> this.matchRegExp = /~?(?:\$):[^\s<>{}\[\]`|'"\\^~]+(?:\/|\b)/mg
~? => Fine
(?:\$) => reducable to \$
: => fine
[^\s<>{}\[\]`|'"\\^~] => Okay, all of those characters are excluded and 
may not occure anywhere in the link
(?:\/|\b) => So the end is either at a / or a word boundary? I think it 
should be reducable to \b as / already denotes a word boundary if 
followed by a word character.

Anyhow: I think I do not fully understand what this is about.